src/scenes/home/header/header.css

```diff
@@ -1,16 +1,21 @@
 .header {
   display: flex;
+  justify-content:space-around; 
   align-items: center;
   height: 80px;
   background-color: rgb(117,129,143);
-  padding: 0 2rem;
+  /*padding: 0 2rem;*/
+  width: 100%;
+  position: -webkit-sticky;
```

Comment:
We have an auto-prefixer built into our webpack build script. You won't need prefixes!